JenkinsConsoleUtility/jcuSrc/Commands/GitApiCommand.cs

```diff
@@ -86,7 +86,7 @@ private class GitHubReleaseRequest : NewRelease
         {
             private const string TagFormat = "{major}.{minor}.{date}";
             private const string NameFormat = "{sdkName} version {major}.{minor}";
-            private const string BodyFormat = "https://api.playfab.com/releaseNotes/#{date}";
+            private const string BodyFormat = "https://learn.microsoft.com/en-us/gaming/playfab/release-notes/#{date}";
```

Comment:
Remove the "en-us". Localization gets set automatically, so the link should just be "https://learn.microsoft.com/gaming/playfab/release-notes/#{date}"